test/acceptance/features/steps/quarkus_application.py

```diff
@@ -0,0 +1,110 @@
+from command import Command
+import re
+import time
+import requests
+from openshift import Openshift
+from pyshould import should_not, should
+
+
+class QuarkusApplication(object):
+
+    cmd = Command()
+
+    quarkus_application = "https://github.com/sbose78/using-spring-data-jpa-quarkus"
+    image_name_with_tag = "ubi-quarkus-native-s2i:19.1.1"
+    api_end_point = '{route_url}/api/status/dbNameCM'
+    openshift = Openshift()
+
+    name = ""
+    namespace = ""
+    build_pod_name_pattern = "{name}-[1-9]\\d*-build"
+    deployment_name_pattern = "{name}-\\w+-deployment"
+
+    def __init__(self, name, namespace):
+        self.name = name
+        self.namespace = namespace
+
+    def install(self):
+        image_stream_output = self.openshift.create_image_stream(self.name, self.namespace)
+        output_match = re.search(r'.*imagestream.image.openshift.io/%s\s(unchanged|created)' % self.name, image_stream_output)
+        output_match | should_not.be_none.desc(f"Image stream creation  mismatch: {image_stream_output}")
+
+        build_config_output = self.openshift.create_build_config(self.name, self.namespace, self.quarkus_application, self.image_name_with_tag)
+        output_match = re.search(r'.*buildconfig.build.openshift.io/%s\s(unchanged|created)' % self.name, build_config_output)
+        output_match | should_not.be_none.desc(f"Build config creation output mismatch: {build_config_output}")
+
+        build_pod_name = self.get_pod_name_running(self.build_pod_name_pattern, wait=True)
+        build_pod_name | should_not.be_none.desc("Running Build pod name output")
+
+        status, output = self.openshift.wait_for_build_pod_status(build_pod_name, self.namespace, wait_for_status="Succeeded")
+        output | should.be_equal_to("Succeeded").desc("Build pod status should be Succeeded")
+
+        knative_service_output = self.openshift.create_knative_service(self.name, self.namespace)
+        output_match = re.search(r'.*service.serving.knative.dev/%s\s(created|configured|unchanged)' % self.name, knative_service_output)
+        output_match | should_not.be_none.desc(f"Knative serving creation output mismatch: {knative_service_output}")
+
+        return True
```

Comment:
what is the point of  return always true? we can drop that then.

Reply:
As `quarkus_app_is_imported_as_knative_service` function from steps.py expects a value for assertion. We have followed this in nodejs as well.